cookbooks/bcpc-hadoop/recipes/cluster_policy_user.rb

```diff
@@ -0,0 +1,17 @@
+user 'bach_policy' do
+  manage_home true
+  comment 'BACH Cluster policy enforcer'
+  only_if { node['bach_policies']['create_local_user'] }
```

Comment:
We tend to use [`user_exist()`](https://github.com/bloomberg/chef-bach/blob/e4352ff2604bf14b19cee62fffb045fc6e321e54/cookbooks/bcpc-hadoop/libraries/utils.rb#L350-L358) for testing if a user exists; can this too?